pyproject.toml

```diff
@@ -47,6 +47,9 @@ doc = [
 [tool.setuptools]
 py-modules = []
 
+[tool.setuptools.packages.find]
+include = ["mdgeom"]
```

Comment:
Would not have thought about that, especially as the sdist seemed to contain all files.

cookie cutter fix...